src/ui/MAVLinkDecoder.h

```diff
@@ -25,7 +25,7 @@ public slots:
     /** @brief Shift a timestamp in Unix time if necessary */
     quint64 getUnixTimeFromMs(int systemID, quint64 time);
 
-    static const size_t cMessageIds = 256;
+    static const size_t cMessageIds = 512;
```

Comment:
Why 512? 
Vendor-specific messages in v2 are allocated higher IDs : https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/uAvionix.xml

We can probably remove this check completely : 
```
if (message.msgid >= cMessageIds) {
       // No support for messag ids above 255
       return;
}
```